### ISSUE TITLE
collaborations API does not seem to be able to list collaborations

### ISSUE THREAD

**Issue by pgoldweic** [instructure/canvas-lms#884]
### Summary:

I am experimenting with the collaborations API but I cannot obtain a listing of collaborations for a given course (note: I've just created a collaboration in the course, so I know there is at least one, and I've enabled collaborations in the course). 
### Steps to reproduce:
1. Pick a published course (e.g. mycourseid) and have an administrator create a collaboration with a particular student in the class (Google Docs).
2. As this same administrator, place the collaborations API call (by typing the following url in the browser): https://mydomain.instructure.com/api/v1/courses/mycourseid/collaborations

I get an empty list as a response
### Expected behavior:

I expected to get a list of collaboration elements (with an undocumented format, by the way). Is this a bug?

**Comment by simonista** [comment]
Hi @pgoldweic, As you can see here: https://github.com/instructure/canvas-lms/blob/stable/app/controllers/collaborations_controller.rb#L99, this api endpoint only returns ExternalToolCollaboration types.  I'm not sure exactly why that is, but I'll at least make a note in the docs for now (I'm also working on documenting the object, see https://github.com/instructure/canvas-lms/issues/895)

**Comment by rivernate** [comment]
Hi @pgoldweic, the future vision of Collaborations is to only use ExternalToolCollaborations moving forward.  All of the other types will eventually be deprecated.  
That endpoint was created specifically for the new UI used for the ExternalToolCollaborations which is why it only returns back those types.

Let me know if you have any questions

**Comment by pgoldweic** [comment]
Thanks for the responses. Seems like it is crucial to then understand what ExternalToolCollaborations are, and in particular to understand what is and is not such a thing (hopefully without going through the source code). I think clear documentation is essential here (when I saw an announcement about a 'collaborations's API' I figured that it would be safe to simply try to use such API in the simplest way I could imagine; this proves otherwise :-(). Looking forward to some real documentation on this API. Thanks @simonista and @revernate !
